src/MolSim.cpp

```diff
@@ -55,17 +69,21 @@ int main(int argc, char *argsv[]) {
   // for this loop, we assume: current x, current f and current v are known
   while (current_time < end_time) {
     // calculate new x
-    calculateX(delta_t);
+    calculations.calculateX(delta_t);
     // calculate new f
-    calculateF();
+    if(fileInputType == 1) {
+    calculations.calculateF();
+    } else {
+      calculations.calculateLJF();
+    }
     // calculate new v
-    calculateV(delta_t);
+    calculations.calculateV(delta_t);
 
     iteration++;
     if (iteration % 10 == 0) {
       plotParticles(iteration, fileType);
```

Comment:
I don't see an option to disable plotting. This would be needed to do performance analysis. For eg. it is not recommended to write the output when doing time measurements. Consider adding an option to disable plotting when needed.